src/lib/JsResolver/authToken.ts

```diff
@@ -0,0 +1,36 @@
+import { Signer } from "@ethersproject/abstract-signer";
+import { SiweMessage } from "siwe";
+
+export const getAuthToken = async (signer: Signer): Promise<string> => {
+  try {
+    const domain = "app.gelato.network";
+    const uri = "http://app.gelato.network/";
+    const address = await signer.getAddress();
+    const version = "1";
+    const chainId = 1;
+    const statement = "Sign this message to upload/fetch JsResolver";
```

Comment:
Let's make the message configurable when we instantiate the JsResolver class
So that the UI could redefine it eventually

Reply:
[0ad911d](https://github.com/gelatodigital/ops-sdk/pull/25/commits/0ad911d23afa6f66aa9a57ebb88a4053d5471f28)

Reply:
[c7685a1](https://github.com/gelatodigital/ops-sdk/pull/25/commits/c7685a1bbd659e487b8598900f90b37794006ae5)